extension-androidauto/build.gradle.kts

```diff
@@ -50,6 +50,6 @@ project.apply {
   from("$rootDir/gradle/ktlint.gradle")
   from("$rootDir/gradle/lint.gradle")
   from("$rootDir/gradle/jacoco.gradle")
-  from("$rootDir/gradle/sdk-registry.gradle")
+//  from("$rootDir/gradle/sdk-registry.gradle")
```

Comment:
maybe not related to this PR, but you can add some flag `publish-android-auto-extension`

Reply:
considering this is short lived configuration, I'm fine with commenting it out